pkg/users/account-create-dialog.js

```diff
@@ -415,21 +519,23 @@
                 {
                     caption: _("Create account with non-unique UID"),
                     style: "warning",
-                    clicked: () => {
-                        return passwd_check(
-                            false, // force weak password was NOT clicked
-                            true, // force user with non-unique UID WAS clicked
-                            state.real_name,
-                            state.user_name,
-                            state.password,
-                            state.password_confirm,
-                            state.locked,
-                            state.change_passw_force,
-                            state.uid,
-                            accounts,
-                            change
-                        );
-                    }
+                    clicked: () => passwd_check(
+                        false, // force weak password was NOT clicked
+                        false, // force user with existing home directory was NOT clicked
+                        true, // force user with non-unique UID was WAS clicked
+                        state.real_name,
+                        state.user_name,
+                        state.password,
+                        state.password_confirm,
+                        state.locked,
+                        state.home_dir,
+                        state.change_passw_force,
+                        state.uid,
+                        accounts,
+                        state.min_uid,
+                        state.max_uid,
+                        change
```

Comment:
These 16 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18234-20230324-000238-1c01f86d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)